public/js/application.js

```diff
@@ -0,0 +1,71 @@
+$(document).ready(function() {
+  // This is called after the document has loaded in its entirety
+  // This guarantees that any elements we bind to will exist on the page
+  // when we try to bind to them
+
+  // See: http://docs.jquery.com/Tutorials:Introducing_$(document).ready()
+
+
+  // trying to add new-comment button functionality here:
+  $('.comment_link').click(function(event){
```

Comment:
nice work so far on making AJAX requests. keep AJAXing to enhance a user's experience while interacting with your app! 

these AJAX requests looks solid to me. continue practicing.